delete.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require "functions.php";
+require "connectToDb.php";
+
+
+
+    $db = getDB();
+    deleteArtwork($db, $_GET['id']);
+    header("Location: index.php");
+    exit;
```

Comment:
Worth making sure that $_GET['id'] is set before you try to use it, just incase someone manually visits delete.php